app/scripts/models/auth_brokers/base.js

```diff
@@ -95,15 +95,6 @@ define(function (require, exports, module) {
       });
     },
 
-    /*
-     * Check if the environment supports the cancelling of the flow.
-     *
-     * @returns {Boolean}
-     */
-    canCancel () {
-      return false;
-    },
-
     /**
```

Comment:
While reviewing the test changes which [remove the tests for cancel](https://github.com/mozilla/fxa-content-server/pull/4374/files#diff-bb855ff272ab5614c711dcc8c4802932L48), it looks like `cancel` below can also be removed.